app/services/acuant/request.rb

```diff
@@ -0,0 +1,68 @@
+module Acuant
+  class Request
+    def path
+      raise NotImplementedError
+    end
+
+    def body
+      raise NotImplementedError
+    end
+
+    def handle_http_response(_response)
+      raise NotImplementedError
+    end
+
+    def method
+      :get
+    end
+
+    def url
+      URI.join(Figaro.env.acuant_assure_id_url, path)
+    end
+
+    def headers
+      {
+        'Authorization' => authorization_header_value,
+        'Accept' => 'application/json',
+      }
+    end
+
+    def send
```

Comment:
can we call this something else? `fetch`, `call`? Because `.send` is the built-in method for calling methods

Reply:
Good catch, will do.